core/src/main/java/org/elasticsearch/indices/breaker/StaticCircuitBreakerService.java

```diff
@@ -26,14 +26,30 @@
 /**
  * Class that returns a breaker that never breaks
  */
-public class NoneCircuitBreakerService extends CircuitBreakerService {
+public class StaticCircuitBreakerService extends CircuitBreakerService {
```

Comment:
> as I said this is just needed for unit tests

If people set the breaker type to none, this class is used and it's important it will return a NoopCircuitBreaker for each type.

PS. just double checking you know you can register an instance with Guice, not only classes.